packages/tabs/src/LionTabs.js

```diff
@@ -65,33 +63,34 @@ function cleanButton({ el, clickHandler, keydownHandler, keyupHandler }) {
 }
 
 /**
- * @param {Element} el
+ * @param {HTMLElement} el
  * @param {boolean} withFocus
  */
 function selectButton(el, withFocus = false) {
   if (withFocus) {
     el.focus();
   }
 
-  el.setAttribute('selected', true);
-  el.setAttribute('aria-selected', true);
-  el.setAttribute('tabindex', 0);
+  el.setAttribute('selected', 'true');
+  el.setAttribute('aria-selected', 'true');
+  el.setAttribute('tabindex', '0');
 }
 
 /**
- * @param {Element} el
+ * @param {HTMLElement} el
  */
 function deselectButton(el) {
   el.removeAttribute('selected');
-  el.setAttribute('aria-selected', false);
-  el.setAttribute('tabindex', -1);
+  el.setAttribute('aria-selected', 'false');
+  el.setAttribute('tabindex', '-1');
 }
 
 /**
- * @param {KeyboardEvent} ev
+ * @param {Event} ev
```

Comment:
EventHandlerNonNull only accepts normal Events, so I'm type casting it to a KeyboardEvent in the function. Alternatively we could create KeyboardEventHandlerNonNull type definition but I'm not sure how to do this in JSDocs, so help is welcome :)